app/views/forever_style_guide/sections/components/navigation/_nav_account_dropdown.erb

```diff
@@ -0,0 +1,35 @@
+<li class='navbar-account_item navbar-account_item-cart' id='mini-cart' data-turbolinks-temporary>
+  <%= link_to cart_url, class: "nav-icon-stack hidden-grid-float-breakpoint" do %>
+    <i class='fa fa-shopping-cart'></i>
+    <span class="nav-icon-label">Cart</span>
+  <% end %>
+</li>
```

Comment:
Cart is defined twice in lieu of more complex DOM order manipulation to account for it's display outside of the offcanvas toggle on mobile. I'll add comments to clarify that in both places in the markup.

The id was in the working version of the store so I think it's best to persist that, assuming it will be used later.